docs/FOLDER_STRUCTURE.md

```diff
@@ -0,0 +1,69 @@
+# Folder structure
+
+This package expects that your data has already been pose tracked and proofread with
+[SLEAP](https://sleap.ai), and organized according to the specifications below.
+
+**It is very important to note that any file ending in an asterisk can be generated from running certain functions in the triangulation and calibration modules. Files and folders not ending in an asterisk must exist before running any functions from sleap-anipose, and must exist in the structure detailed below.** 
```

Comment:
Maybe it's better to have two example folder structures: one with just the initial files at the start of the workflow (before any sleap-anipose steps), and one with everything, including the outputs at the end of the pipeline.

Reply:
You are absolutely right. I'll start with just explaining the basics of what a session folder looks like and then expand from there. That's the key to the organization anyways.

Reply:
Folder structure to be updated in subsequent PR.